### ISSUE TITLE
add progress of loop using cli

### ISSUE THREAD

**Issue by catalamarti** [oxford-pharmacoepi/CohortSymmetry#97]

**Comment by KimLopezGuell** [comment]
For which functions would this be necessary? generateSequenceCohortSet() and summariseSequenceRatio()? Any other? I presume this would only be the case when we have multiple index or marker cohorts. @catalamarti

**Comment by KimLopezGuell** [comment]
Coming back to this, @xihang-chen ?

**Comment by xihang-chen** [comment]
> Coming back to this, @xihang-chen ?

Yes please! @KimLopezGuell , and I think it's good enough doing it only for generateSequenceCohortSet() function, like you said it's for index cohorts and/or marker cohorts having more than one cohort_definition_id. Thanks! :)
